x/programs/rust/examples/single_nft/src/example.rs

```diff
@@ -0,0 +1,147 @@
+use wasmlanche_sdk::simulator::{
+    Client, Endpoint, ExecuteResponse, Key, Param, ParamType, Plan, Step,
+};
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+const PROGRAM_ID: &str = "0000000000000000000000000000000000000000000000001";
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+fn initialize_plan<'a>(
+    nft_name: &'a str,
+    nft_name_length: &'a str,
+    nft_symbol: &'a str,
+    nft_symbol_length: &'a str,
+    nft_uri: &'a str,
+    nft_uri_length: &'a str,
+) -> Plan<'a> {
+    let steps = vec![
+        Step {
+            description: "create program",
+            endpoint: Endpoint::Execute,
+            method: "program_create",
+            params: vec![Param {
+                name: "program_path",
+                param_type: ParamType::String,
+                value: "../../examples/testdata/single_nft.wasm",
+            }],
+            require: None,
+        },
+        Step {
+            description: "init",
+            endpoint: Endpoint::Execute,
+            method: "init",
+            params: vec![
+                Param {
+                    name: "program_id",
+                    param_type: ParamType::Id,
+                    value: PROGRAM_ID,
+                },
+                Param {
+                    name: "nft_name",
+                    param_type: ParamType::String,
+                    value: nft_name,
+                },
+                Param {
+                    name: "nft_name_length",
+                    param_type: ParamType::U64,
+                    value: nft_name_length,
```

Comment:
might be nice if the SDK include a method that would add this second param for you

Reply:
Agreed